xml/art_admin_slemicro_cockpit.xml

```diff
@@ -125,945 +139,2118 @@
    address in your web browser:
   </para>
 <screen>https://<replaceable>IP_ADDRESS_OF_MACHINE</replaceable>:9090</screen>
-
-
-
  </section>
-
-<section xml:id="sec-cockpit-authentication">
+ <section xml:id="sec-cockpit-authentication">
   <title>Cockpit authentication</title>
   <section xml:id="sec-authentication-login">
-    <title>Authentication</title>
+   <title>Authentication</title>
+   <para>
+    Cockpit enables you to log in directly to each machine that can expose the
+    9090 port. If the port cannot be accessed on the particular machine, you
+    can still use Cockpit to administer the machine by using it as a secondary
+    server. For a procedure of adding a server as secondary, refer to
+    <xref linkend="sec-adding-servers"/>.
+   </para>
+   <note>
+    <title>A limited count of secondary servers</title>
     <para>
-      Cockpit enables you to log in directly to each machine that can expose the 9090 port. If the port cannot be accessed on the particular machine, you can still use Cockpit to administer the machine by using it as a secondary server. For a procedure of adding a server as secondary, refer to <xref linkend="sec-adding-servers"/>. 
+     The count of secondary servers that you can administer from one primary
+     session is limited to 20. If you need to administer more servers, add
```

Comment:
one more "session" -> "server"

Reply:
I'll fix it along with the proofread :)

Reply:
Ah, if there's a followup proofreading round after that then I'm all fine with the contents of technical level.